rust/Makefile.am

```diff
@@ -61,7 +61,8 @@ CARGO_ENV +=	LOCALSTATEDIR=$(e_localstatedir)
 all-local: Cargo.toml
 	mkdir -p $(abs_top_builddir)/rust/gen
 	cd $(abs_top_srcdir)/rust && \
-		CC=$(CC) $(CARGO_ENV) \
+		CC="$(CC)" AR="$(AR)" RANLIB="$(RANLIB)"\
```

Comment:
I do not understand this bash magic env variable setting CC to CC, but it seems to work 🤷